extensions/src/main/java/org/pentaho/platform/web/http/filters/PentahoWebContextFilter.java

```diff
@@ -591,5 +682,9 @@ IPluginManager getPluginManager() {
   IUserSettingService getUserSettingsService() {
     return PentahoSystem.get( IUserSettingService.class, getSession() );
   }
+
+  ConfigurationAdmin getOSGIConfigurationAdmin() {
```

Comment:
Shouldn't it be written as `getOsgiConfigurationAdmin`? Or the other written this way be changed to upper case. Not sure which is the Java convention for acronyms.